Makefile

```diff
@@ -3,15 +3,18 @@ export
 SHELL := /bin/bash
 LOCAL := $(PWD)/usr
 PATH := $(LOCAL)/bin:$(PATH)
-TESSDATA =  $(LOCAL)/share/tessdata
+TESSDATA_BEST = ~/tessdata_best
```

Comment:
Not sure if we should have locations in `$HOME` as default. Since most people install tesseract and its models directly via their Linux distribution, I opt for `/usr/share/tesseract-ocr/4.00/tessdata/` (cf. https://packages.debian.org/buster/all/tesseract-ocr-ces/filelist).

Reply:
I will defer to @stweil for best way to handle this.